mpmath/tests/test_basic_ops.py

```diff
@@ -557,3 +557,14 @@ def get_remainder(x, bits):
 
 def test_issue_260():
     assert mpc(str(mpc(1j))) == mpc(1j)
+
+
+def test_issue_751():
+    x = mp.autoprec(mp.matrix, catch=ZeroDivisionError)([[1e210, 300*1e230], [0, 340*1e230]] )
```

Comment:
Perhaps, just use float literals?
```suggestion
    x = mp.autoprec(mp.matrix, catch=ZeroDivisionError)([[1e210, 3e+232], [0, 3.4e232]] )
```